backend/authentication/locale/fr/LC_MESSAGES/django.po

```diff
@@ -42,51 +42,52 @@ msgstr "Je m'engage à respecter les conditions d'utilisation"
 msgid "Next url"
 msgstr "URL de redirection"
 
-#: authentication/models.py:10
+#: authentication/models.py:9
 msgid "Organization"
 msgstr "Organisation"
 
-#: authentication/models.py:14
+#: authentication/models.py:12
 msgid "TOKEN"
 msgstr "Jeton d'accès à l'API"
 
-#: authentication/views.py:33
+#: authentication/views.py:31
 msgid "Invalid json in the POST request"
 msgstr "Json invalide dans la requête POST"
 
-#: authentication/views.py:42
-msgid "You seem already registered. Try to use the Connection form."
-msgstr "Vous semblez déjà inscrit(e). Essayez d'utiliser l'onglet Connexion."
-
-#: authentication/views.py:51 authentication/views.py:89
+#: authentication/views.py:39 authentication/views.py:84
 #, python-format
 msgid "The link is valid for %d min"
 msgstr "Le lien de connexion est valide pendant %d min"
 
-#: authentication/views.py:53 authentication/views.py:91
+#: authentication/views.py:41 authentication/views.py:86
 msgid "The link does not expire"
 msgstr "Le lien n'a pas de date d'expiration"
 
-#: authentication/views.py:63
+#: authentication/views.py:53
+#, fuzzy
+#| msgid "You seem already registered. Try to use the Connection form."
```

Comment:
You should remove the fuzzy tag or the translation won't be compiled.

Reply:
:+1: I admit I don't really take time to manage the comments in the po file...